dashboard/key-visualizer.md

```diff
@@ -0,0 +1,181 @@
+---
+title: 流量可视化页面
+category: how-to
+aliases: ['/docs-cn/dev/how-to/monitor/key-visualizer/','/docs-cn/dev/key-visualizer-monitoring-tool/']
+---
+
+# 流量可视化页面
+
+流量可视化页面 (Key Visualizer) 可用于分析 TiDB 集群的使用模式和排查流量热点。该页面可视化地呈现了 TiDB 集群一段时间的流量情况。
+
+## 访问页面
+
+可以通过以下两种方法访问 Key Visualizer 流量可视化页面：
+
+* 登录 TiDB Dashboard 后，点击左侧导航条的 **Key Visualizer**（流量可视化）：
```

Comment:
Do we have a unified format for these references? In https://github.com/pingcap/docs-cn/pull/3274 they are "「xxx」"

Reply:
In the interface, the button is in English, not Chinese. In our documentation, we can use **bold** to indicate a button.

Reply:
In Chinese UI the link is Chinese（流量可视化）, do we have some preferences?

Reply:
The snapshot image is English only for easier I18N. The Chinese user will see a Chinese UI in most cases.

Reply:
In this Chinese document, it would be better to give an image example of  **Chinese** UI and refer to these buttons in their Chinese names. If an English UI is given in a Chinese doc, I think **we should refer the button as the UI shows.**

Reply:
We don't have so much time to produce every screenshots in every supported language.